src/java/map/Material.java

```diff
@@ -22,8 +23,8 @@ public Material(String environment, String texture, String normal, float texture
         this(environment, environment, texture, normal, textureScale, normalScale);
     }
     public Material(String texEnv, String normalEnv, String texture, String normal, float textureScale, float normalScale) {
-        texturePath = String.format(pathFormat, texEnv, texture, "albedo");
-        normalPath = String.format(pathFormat, normalEnv, normal, "normals");
+        texturePath = String.format(texturePathFormat, texEnv, texture, "albedo");
+        normalPath = String.format(normalPathFormat, normalEnv, normal);

```

Comment:
@n3roxis pls verify this

Reply:
verify what ?
i've updated the template generator that I wrote myself aswell as this part of the code that used the same layout as the template generator and updated both (aswell as all the templates) according to it
Look at the screenshots in the PR
and look at the JSON files aswell